cmds/dhclient/dhcp6client/dhcp6client_test.go

```diff
@@ -0,0 +1,329 @@
+// Command dhcp6d is an example DHCPv6 server.  It can only assign a
```

Comment:
Where should we put the copyright announcement? At the top of the code, or right before the parts that is written by mdlayher?